docs/docs/user-guide/appendix/FAQ.md

```diff
@@ -0,0 +1,40 @@
+# Frequently Asked Questions
+
+### Is BioNeMo free to use?
+
+Yes, BioNeMo is free to use.
```

Comment:
Minor -- maybe mention that we use an Apache 2 license for everything in the codebase (could even link to the LICENSE file). _Putting on nerd hat_ we're both free as in beer and free as in freedom :)